routes/api/household.js

```diff
@@ -59,6 +59,41 @@ router
       });
   });
 
+// resolves as /api/household/:id    
+router
+.route("/:id")
```

Comment:
I'm concerned that this could catch other routes, since it's looking for any text after the root address. I think there should be something before the `/:id` to avoid other routes getting caught by this one and their route info passed as `id`.